### ISSUE TITLE
Missing state flag in stdout in some cases

### ISSUE THREAD

**Issue by ccztux** [NagiosEnterprises/ncpa#575]
There are some cases where stdout has no state flag:
```
18:36:16 [nagios@localhost]:~/Dokumente/git-repos/ncpa/client$ ./check_ncpa.py -H 10.0.0.73 -t mytoken -M '/plugins/test'
The plugin (test) requested does not exist. You may be trying to access the 'test.sh' node.
```

```
18:36:45 [nagios@localhost]:~/Dokumente/git-repos/ncpa/client$ ./check_ncpa.py -H 10.0.0.73 -t mytoken -M '/system/agent_versio'
The node (agent_versio) requested does not exist. You may be trying to access the 'agent_version' node.
```

Can you please add the state flag for consistent output format?

**Comment by ccztux** [comment]
If #574 will be merged, it would be great, if the state flag could also be added to error messages like described in this [issue comment](https://github.com/NagiosEnterprises/ncpa/issues/573#issuecomment-531312561).

**Comment by ccztux** [comment]
Another case where the state flag is missing:
```
20:11:54 [nagios@localhost]:~/Dokumente/git-repos/ncpa/client$ ./check_ncpa.py -H 10.0.0.73 -t mytoken -M '/plugins' 
Unable to run check on node without check method. Requested 'plugins' node.
```

**Comment by jomann09** [comment]
I fixed these two cases with 2.1.10, if there are any more that you find let me know. There are a few messages (like the invalid credentials message) that won't be updated but any others let me know!

**Comment by jomann09** [comment]
Just an update, this'll be in 2.2.0, which will be the next version since we have a lot more updates than I thought.
